CMakeLists.txt

```diff
@@ -131,4 +141,65 @@ configure_lit_site_cfg(
   ${CMAKE_CURRENT_SOURCE_DIR}/lit.cfg.py
 )
 
-ie_cpack(BENCHMARK_APP)
+# ie_cpack(BENCHMARK_APP)
+
+
+
+
+
+# set(BIN plaidml_edsl_tests_cc_test)
+# install(
+#   TARGETS ${BIN} 
+#   DESTINATION bin 
+#   COMPONENT DEVKIT
+# )
+
+# include(GetPrerequisites)
+# set(BINARY_LOCATION "${CMAKE_CURRENT_BINARY_DIR}/bin/${BIN}")
+# get_prerequisites(${BINARY_LOCATION} DEPENDENCIES 0 0 "" "")
+
+# foreach(DEPENDENCY_FILE ${DEPENDENCIES})
+#   install(
+#     FILES ${DEPENDENCY} 
+#     DESTINATION lib 
+#     COMPONENT DEVKIT
+#   )
+# endforeach()
+
+install(
+    TARGETS plaidml_edsl_tests_cc_test
+    DESTINATION bin
+    COMPONENT DEVKIT
+)
+
+install(
+    PROGRAMS 
+      ${LLVM_BINARY_DIR}/lib/libomp.so
+      ${LLVM_BINARY_DIR}/lib/libmlir_runner_utils.so.12git
+      /lib/x86_64-linux-gnu/libpthread.so.0     
+      /lib/x86_64-linux-gnu/libpthread-2.27.so  
+      /lib/x86_64-linux-gnu/librt.so.1  
+      /lib/x86_64-linux-gnu/librt-2.27.so
+      /lib/x86_64-linux-gnu/libdl.so.2
+      /lib/x86_64-linux-gnu/libdl-2.27.so
+      /lib/x86_64-linux-gnu/libz.so.1
+      /lib/x86_64-linux-gnu/libz.so.1.2.11
+      /lib/x86_64-linux-gnu/libtinfo.so.5
+      /lib/x86_64-linux-gnu/libtinfo.so.5.9
```

Comment:
We don't want to package up the binaries that are part of the machine doing the building.